### PR TITLE
Add API test python script - Push chart by Helm3 registry/chart CLI

### PR DESCRIPTION
**Pull request by danfengliu**
Signed-off-by: danfengliu <danfengl@vmware.com>

Please reference issue: #10935 

Fixed issue: #10762,#10763

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -130,6 +130,9 @@ jobs:
           sudo cp ./tests/harbor_ca.crt /usr/local/share/ca-certificates/
           sudo update-ca-certificates
           sudo service docker restart
+          wget https://get.helm.sh/helm-v3.0.0-linux-386.tar.gz && tar zxvf helm-v3.0.0-linux-386.tar.gz && \
```
Comment (wy65701436) [inline .github/workflows/CI.yml]: use 3.1 instead?

File: tests/apitests/python/test_push_chart_by_helm3_chart_cli.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+        #3. Push an chart(CA) to Harbor by helm3 registry/chart CLI successfully;
+        chart_cli_ret = library.helm.helm_chart_push_to_harbor(self.chart_file, self.archive,  harbor_server, TestProjects.project_push_chart_name, self.repo_name, self.verion, user_name, self.user_push_chart_password)
+        print "chart_cli_ret:", chart_cli_ret
+
+        #4. Get chart(CA) from Harbor successfully;
+        artifact = self.artifact.get_reference_info(TestProjects.project_push_chart_name, self.repo_name, self.verion, **TestProjects.USER_CLIENT)
+        print "artifact:", artifact
+
+        #5. TO_DO: Verify this chart artifact information, like digest;
```
Comment (wy65701436) [inline tests/apitests/python/test_push_chart_by_helm3_chart_cli.py]: can we just add the checking on "values.yaml, dependencys, readme.md"?

File: .github/workflows/CI.yml (hunk 1 above)
Comment (danfengliu) [reply .github/workflows/CI.yml]: Yes, helm 3.1.1 has replaced V3.0.0 in CI environment.
 
"Saving to: ‘helm-v3.1.1-linux-386.tar.gz’"

File: tests/apitests/python/test_push_chart_by_helm3_chart_cli.py (hunk 2 above)
Comment (danfengliu) [reply tests/apitests/python/test_push_chart_by_helm3_chart_cli.py]: I will start a new PR for tracking this verification criteria.

**Pull request merged**